client/src/components/MultiJurisdictionAudit/index.tsx

```diff
@@ -135,6 +141,15 @@ export const JurisdictionAdminView: React.FC = () => {
             uploadBallotManifest={uploadBallotManifest}
             deleteBallotManifest={deleteBallotManifest}
           />
+          {ballotManifest.processing &&
+            ballotManifest.processing.status ===
+              FileProcessingStatus.PROCESSED && (
+              <BatchTallies
+                batchTallies={batchTallies}
+                uploadBatchTallies={uploadBatchTallies}
+                deleteBatchTallies={deleteBatchTallies}
+              />
+            )}
```

Comment:
it might be clearer for the user if the batch tallies form is always displayed but disabled until the ballot manifest is uploaded. I'm not sure, but I have a hunch that they will want to know that they need to generate both files at the beginning, rather than generate one and then find out they have to generate the other